static/js/modules/game/articleRenderer.js

```diff
@@ -86,12 +81,16 @@ export class ArticleRenderer {
             });
 
             // Remove "/wiki/" from string
-            this.loadPageWrapper(linkEl.getAttribute("href").substring(6))
+            this.loadPage(linkEl.getAttribute("href").substring(6))
         }
     }
 }
 
-
+function setMargin(frame) {
```

Comment:
Depending on how we choose to format mobile, the timebox element's ID might not always be "time-box"